charts/simple-app/Chart.yaml

```diff
@@ -13,5 +13,5 @@ dependencies:
     repository: https://k8s-charts.nextdoor.com
     condition: istio-alerts.enabled
   - name: nd-common
-    version: 0.0.4
+    version: 0.0.5
```

Comment:
You'll need to simultaneously update `daemonset-app` and `stateful-app` because both of those charts are also using `repository: file://../nd-common`. When we use the `file://...` reference, then you MUST update the version on all of the places where its used... I  hate it, but it's sort of a good forcing function to make sure we keep all the charts in-line.